src/components/comment/CommentVoteForm.tsx

```diff
@@ -1,72 +1,75 @@
 'use client';
+import React, { useCallback, memo } from 'react';
 import { voteHandlerAction } from '@/actions/commentVote';
 import { useAction } from '@/hooks/useAction';
 import { VoteType } from '@prisma/client';
-import { ThumbsDownIcon, ThumbsUpIcon } from 'lucide-react';
+import ThumbUpOffAltOutlinedIcon from '@mui/icons-material/ThumbUpOffAltOutlined';
```

Comment:
do we need mui icons or its just another library? lucid was fine i think

Reply:
Lucid is good. But I did not find any outlined icons so i added mui icons( which is wrong )